.github/workflows/slack-notify.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+# Runs your workflow anytime the release event occurs
+name: Slack Notification
+on:
+  pull_request:
+    branches:
+      - master
+      - development
+  release:
+    types: [published]
+
+jobs:
+
+  slack_notification_job:
+
+    name: Slack Notification
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Check out code
+        uses: actions/checkout@master
+
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@master
+        env:
+          SLACK_CHANNEL: github-ci
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+          SLACK_USERNAME: github-actions-boy
+          SLACK_ICON: https://avatars3.githubusercontent.com/in/15368?s=48
+          SLACK_TITLE: New Release https://github.com/pasientskyhosting/ps-otu-ldap
```

Comment:
Should slack notifications be part of the standard action lib instead?

And you would then be able to overwrite the `SLACK_CHANNEL` in case you want a project to notifiy another chanel?